internal/schema/schema.go

```diff
@@ -384,8 +354,46 @@ func (s *Schema) GetQueryStringForEndpoint(typePath []*Type, fieldPath []string,
 	return final
 }
 
+// GetQueryArg returns the GraphQL formatted Query Argument
+// including formatting for List and NonNull
+//
+// Example:
+// {
+//   "name": "tags",
+//   "type": { "kind": "NON_NULL", "ofType": { "kind": "LIST", "ofType": { "kind": "NON_NULL", "ofType": { "name": "TaggingTagInput", "kind": "INPUT_OBJECT" } } } }
+// }
+//   [TaggingTagInput!]!
+//
+func (s *Schema) GetQueryArg(field Field) QueryArg {
```

Comment:
Can we include a test for this method if its not already covered?

Reply:
It is not directly tested, but is covered via `TestSchema_BuildQueryArgsForEndpoint`